ras_backstage/controllers/collection_instrument_controller.py

```diff
@@ -14,15 +15,40 @@ def upload_collection_instrument(survey_id, collection_exercise_id, file):
     url = f'{app.config["RAS_COLLECTION_INSTRUMENT_SERVICE"]}' \
           f'collection-instrument-api/1.0.2/upload/{collection_exercise_id}'
 
-    classifiers = {}
-    if survey_id:
-        classifiers['SURVEY_ID'] = survey_id
-    if collection_exercise_id:
-        classifiers['COLLECTION_EXERCISE'] = collection_exercise_id
+    classifiers = _build_classifiers(collection_exercise_id, survey_id)
 
     files = {"file": (file.filename, file.stream, file.mimetype)}
     request_handler(url=url, method='POST', auth=app.config['BASIC_AUTH'], files=files,
```

Comment:
We need a status code check here. 
Currently even if we fail to upload the collection instrument it shows as successful in the UI